e2e/patch_from_repo/MODULE.bazel

```diff
@@ -0,0 +1,37 @@
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(name = "aspect_rules_js", version = "0.0.0")
+bazel_dep(name = "local_repo", version = "0.0.0")
+
+local_path_override(
+    module_name = "aspect_rules_js",
+    path = "../..",
+)
+
+local_path_override(
+    module_name = "local_repo",
+    path = "./local_repo"
+)
+
+npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm")
+
+npm.npm_translate_lock(
+    name = "npm",
+    data = [
+        "//:package.json",
+        "//:pnpm-workspace.yaml",
+    ],
+    patch_args = {
+        "meaning-of-life": ["-p1"],
+    },
+    patches = {
+        # TODO: Because `patches` is a string_list_dict that doesn't take labels, we can't
+        # pass in a label like `@local_repo//:....` because only the root repository knows
+        # how to map this repo to its local override. This causes rules_js to fail when trying
+        # to convert the label into a path.
+        "meaning-of-life@1.0.0": ["@@local_repo~override//:meaning-of-life@1.0.0.patch"],
+    },
```

Comment:
@gregmagolan This was a fun one... Because `patches` doesn't take labels, we lose the repo mapping to the local override and have to use an absolute label like this.

Reply:
🏋️